### ISSUE TITLE
[16.0] base: model_terms are not migrated

### ISSUE THREAD

**Issue by diggy128** [OCA/OpenUpgrade#4544]
Migration scripts for base module

model_terms in v15 are not migrated in v16.0 

I looked into the new way translations work, but does this mean we have to re-translate (the modified terms) in all the views and reports for v16?

**Comment by pedrobaeza** [comment]
I don't get what is `model_terms`. It's not a table nor a model. Can you be more specific?

**Comment by diggy128** [comment]
@pedrobaeza Sorry I should have been clearer.
model_terms is the type used for the selection in ir.translation that signifies the term of translation. 
This is the one used for translating xml (view and qweb excerpts) up until v15.0

**Comment by pedrobaeza** [comment]
This is not something of OpenUpgrade, but Odoo, which has changed the way translations work, and they don't allow customization per BD of these terms. You have to translate them in the .po files, and they are loaded in memory on Odoo startup.

I personally think that customizing terms per DB is not a good practice.

Closing as not something of OpenUpgrade.

**Comment by diggy128** [comment]
@pedrobaeza 
I've already looked into how translation handling has changed, thus I've opened this issue.

If we run an upgrade with the existing OpenUpgrade scripts, we lose all _customizations_ in translations (fair enough, should be done in .po files) BUT we also lose all translations in website pages and ANY html field as those were saved in ir.translation with type `model_terms` and OpenUpgrade scripts ignore that type (not implemented yet).

So, I kindly request to reopen the issue so that there won't be any surprises for anyone migrating, only to find out that their translations are long gone!
